oxidation/libparsec/crates/types/src/time.rs

```diff
@@ -158,6 +169,10 @@ mod time_provider {
             let now = chrono::Utc::now();
             now.into()
         }
+
+        pub async fn sleep(&self, time: std::time::Duration) {
+            tokio::time::sleep(time).await
+        }
     }
 }
 
```

Comment:
```suggestion

#[cfg(not(target_arch = "wasm32"))]
```
This code will not compile on `wasm32`

Reply:
fixed